plugins/github-pull-requests-board/src/api/useGetPullRequestsFromRepository.ts

```diff
@@ -26,27 +26,41 @@ export const useGetPullRequestsFromRepository = () => {
     async (repo: string): Promise<PullRequestsNumber[]> => {
       const [organisation, repositoryName] = repo.split('/');
 
-      const { repository } = await graphql(
-        `
-          query ($name: String!, $owner: String!) {
-            repository(name: $name, owner: $owner) {
-              pullRequests(states: OPEN, first: 10) {
-                edges {
-                  node {
-                    number
+      const pullRequestEdges = [];
+      let result: GraphQlPullRequests<PullRequestsNumber[]> | undefined =
+        undefined;
+      do {
+        result = await graphql(
+          `
+            query ($name: String!, $owner: String!, $endCursor: String) {
+              repository(name: $name, owner: $owner) {
+                pullRequests(states: OPEN, first: 100, after: $endCursor) {
```

Comment:
Would it make more sense to use a prop here? That could be passed in like say this: `<EntityTeamPullRequestsCard  defaultLimit='250' />`. Then if 100 is too little or too much it can easily be changed.

Reply:
I like this suggestion! 👍

Reply:
Thanks for the input! I implemented the optional prop for the components, let me know if you see any issues. @awanlin  @jhaals